src/Components/VoteIncrement.js

```diff
@@ -0,0 +1,46 @@
+import { apiGetArticlesByArticleId } from "../Utils/api";
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+const VoteIncrement = () => {
+  const [vote, setVotes] = useState(0);
+  let { article } = useParams();
+
+  useEffect(() => {
+    apiGetArticlesByArticleId(article).then((response) => {
+      setVotes(response.articles.votes);
+    });
+  }, [article]);
+
+  const incrementVote = () => {
+    setVotes(vote + 1);
+  };
+
```

Comment:
This will only update the votes in the current view and is not updating on the backend. You will need to make a patch request to your api in order to deal with this. Remember that if the api call fails, then we will need to reverse any optimistic rendering.